tests/documentation/test_files/DocumentationStyleBear/bad_file4.py.test

```diff
@@ -0,0 +1,9 @@
+def myfunc(a, b, x):
```

Comment:
can you add a test case in bad_file4 where the def docstring doesnt start with a blank new line. i.e.
```
def ....
    """
    :param ...
```